barretenberg/cpp/src/barretenberg/plonk/composer/standard_composer.hpp

```diff
@@ -54,13 +56,12 @@ class StandardComposer {
         };
         return result;
     }
-    std::shared_ptr<plonk::proving_key> compute_proving_key(const CircuitBuilder& circuit_constructor);
-    std::shared_ptr<plonk::verification_key> compute_verification_key(const CircuitBuilder& circuit_constructor);
+    std::shared_ptr<plonk::proving_key> compute_proving_key(CircuitBuilder& circuit_constructor);
```

Comment:
const removed because compute_proving_key also computes witness data and this involves changing builder members (at least `lookup_gates`, maybe others)